lib/grim/image_magick_processor.rb

```diff
@@ -48,9 +48,18 @@ def save(pdf, index, path, options)
         command_env['PATH'] = "#{File.dirname(@ghostscript_path)}#{File::PATH_SEPARATOR}#{ENV['PATH']}"
       end
 
-      result, status = Open3.capture2e(command_env, command.join(' '))
+      log(command_env, command)
```

Comment:
I'd probably just inline the log method code here since it is only used once. I also wonder if it wouldn't be useful to log the result/status of the command.